task/01-strings-tasks.js

```diff
@@ -84,7 +80,7 @@ function extractNameFromTemplate(value) {
  *   'cat'       => 'c'
  */
 function getFirstChar(value) {
-    throw new Error('Not implemented');
+  return value.charAt(0);
```

Comment:
You can just use value[0], it’s shorter or if you need a char from the end use .at()

Reply:
Okay, thanks, I'll correct